electrum/base_wizard.py

```diff
@@ -749,9 +765,14 @@ def choose_seed_type(self, message=None, choices=None):
             advanced_choices = [
                 ('create_standard_seed', _('Legacy')),
             ]
+        hint = ' '.join([
+                _("'Legacy' is the original address type, while 'Segwit' is the newer address format with lower fees.") + '\n',
```

Comment:
Why `+ '\n'` ? Wouldn't be more suitable, in this case, use it as a separate list element?

Reply:
done